lsp.el

```diff
@@ -965,6 +965,13 @@ If WORKSPACE is not provided current workspace will be used."
 
 (defalias 'lsp-send-notification 'lsp--send-notification)
 
+(cl-defun lsp-notify (method params &key single)
```

Comment:
Instead of single at this point the pattern is 

(with-current-workspace workspace
  (lsp--send-notification)
...
)